docs/custom_orders.md

```diff
@@ -1,44 +1,45 @@
 # Custom order examples
+Directly ask real humans to classify your data. This guide will show you how to create a classification order using the Rapidata API.
```

Comment:
This file is now for the "advanced" user, right? It should be clear that we are asking real humans at this point no?

Reply:
yes, but i don't think it hurts? what would you rather have there?